packages/themes/src/ThemeContext/Consumer.js

```diff
@@ -16,6 +29,9 @@ export default class Consumer extends Component {
       this.context.themeId && this.context.themeClass
         ? this.context
         : HIGLightTheme;
-    return this.props.children(theme);
+    const proxy = new Proxy(theme, themeDataProxyHandler);
+    const themeArgument =
+      process.env.NODE_ENV === "development" ? proxy : theme;
```

Comment:
Let's change the logic to `process.env.NODE_ENV !== "production"`. This way the error message will be displayed during testing. In fact, we're already seeing value with this displaying error messages for the broken `Input` styles.

Reply:
Sounds good, I was actually going to bring this up during our Daily as I originally had that but wasn't sure if we want it to show during testing, staging, etc.